spring/src/main/java/org/axonframework/spring/config/SpringAxonAutoConfigurer.java

```diff
@@ -350,7 +350,7 @@ private void registerSagaBeanDefinitions(EventProcessingConfigurer configurer) {
             Class<A> aggregateType = (Class<A>) aggregate.getKey();
             Aggregate aggregateAnnotation = aggregateType.getAnnotation(Aggregate.class);
             AggregateConfigurer<A> aggregateConf = AggregateConfigurer.defaultConfiguration(aggregateType);
-            aggregate.getValue().keySet().forEach(aggregateConf::registerSubtype);
+            aggregate.getValue().keySet().forEach(aggregateConf::withSubtype);
```

Comment:
If withSubtypes would accept a Collection of vararg, then the aggregate.getValue().getKeySet() could be added immediately, and it wouldn't break the fluent interface.